Fuzzing failure

Daily slide fuzzing failed. Last 30 lines of the output log are as follows:

```
   Compiling textwrap v0.11.0
   Compiling libfuzzer-sys v0.3.3
   Compiling annotate-snippets v0.9.0
   Compiling atty v0.2.14
   Compiling clap v2.33.2
   Compiling libslide v0.0.1 (/home/runner/work/slide/slide/libslide)
   Compiling slide v0.0.1 (/home/runner/work/slide/slide/slide)
   Compiling slide-fuzz v0.0.0 (/home/runner/work/slide/slide/slide/fuzz)
error[E0432]: unresolved import `slide::OutputForm`
 --> fuzz_targets/slide.rs:3:30
  |
3 | use slide::{run_slide, Opts, OutputForm};
  |                              ^^^^^^^^^^ no `OutputForm` in the root

error[E0560]: struct `slide::Opts` has no field named `output_form`
 --> fuzz_targets/slide.rs:8:9
  |
8 |         output_form: OutputForm::Pretty,
  |         ^^^^^^^^^^^ `slide::Opts` does not have this field
  |
  = note: available fields are: `program`, `emit_format`, `parse_only`, `expr_pat`, `no_emit`

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0432, E0560.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `slide-fuzz`.

To learn more, run the command again with --verbose.
Error: failed to build fuzz script: "cargo" "build" "--manifest-path" "/home/runner/work/slide/slide/slide/fuzz/Cargo.toml" "--target" "x86_64-unknown-linux-gnu" "--release" "--bin" "slide"
```

Download this test case at 

[Command run output][1]

[1]: https://github.com/yslide/slide-workflows/actions/runs/199529043

--- a/.travis.yml
+++ b/.travis.yml
@@ -43,12 +43,17 @@ jobs:
     - rust: nightly
 
 script:
+  # Ensure slide builds
   - cargo build --verbose --all
   - scripts/check test
   - rustup component add clippy || cargo install --git
     https://github.com/rust-lang/rust-clippy/ --force clippy
   - rustup component add rustfmt
   - scripts/check lintformat
+  # Ensure fuzzer builds
+  - pushd slide/fuzz
+  - cargo build
+  - popd
 
 after_success:
   - |

--- a/slide/fuzz/Cargo.toml
+++ b/slide/fuzz/Cargo.toml
@@ -14,6 +14,9 @@ libfuzzer-sys = "0.3"
 [dependencies.slide]
 path = ".."
 
+[dependencies.libslide]
+path = "../../libslide"
+
 # Prevent this from interfering with workspaces
 [workspace]
 members = ["."]

--- a/slide/fuzz/fuzz_targets/slide.rs
+++ b/slide/fuzz/fuzz_targets/slide.rs
@@ -1,11 +1,12 @@
 #![no_main]
 use libfuzzer_sys::fuzz_target;
-use slide::{run_slide, Opts, OutputForm};
+use libslide::EmitFormat;
+use slide::{run_slide, Opts};
 
 fuzz_target!(|program: String| {
     run_slide(Opts {
         program,
-        output_form: OutputForm::Pretty,
+        emit_format: EmitFormat::Pretty,
         parse_only: true,
         expr_pat: false,
         no_emit: true,

